README.md

```diff
@@ -1,2 +1,21 @@
+---
+output: html_document
+---
 # hw01
 Repository for first CFSS homework assignment
+
+# Introduction
+I am a graduate student in the _MAPSS_ program with an economics concentration. 
+The list of classes that I am taking this quarter include:
+
+1. Firm Dynamics and Innovation
+2. Analysis of Micro Data
+3. This Class
+
+You can learn more about the MAPSS program through its website:
+https://mapss.uchicago.edu/
+
+# Report
+I created a new project from version control and pulled from github using git pull command. Then I modified the markdown file in Rstudio. 
+
+The hard problem is to find out the R Markdown code seems to require blank line before the list in order for list syntax to work properly. Trying to understand how commit and push work is also the harder part. The other parts are relatively easy. 
```

Comment:
Yup. It's a bit of a quirk in Markdown, but that's just how it works. Also note that in the future this feedback should be included in the summary of the pull request, not in the actual assignment repo